src/tests/3-corax+.8o

```diff
@@ -111,13 +121,32 @@
   x2 := 27
   y := 1
   drawop im2 imX
+	
+	v0 := 0
+	# call subroutine, which sets v0 to 1, and displays a checkmark
```

Comment:
Maybe I'm thinking about this too simply, but can't we just use flow-control for both tests? Do we need to keep this state?

Reply:
The idea for that was for if 2NNN were to fail, it could see that occur, and skip over 00EE properly. Sometimes, someone could have 00EE implemented, but not 2NNN, and in which case the interpreter would jump to $000. I wanted get the test to continue so the dev could see anything else that failed.

It could be considered unnecessary, but I'll leave that up to you to decide if I should remove it or not.

Reply:
No you're right... I've played with this a bit and it is indeed needed.